CHANGELOG-webpack-dev-server.md

```diff
@@ -0,0 +1,3 @@
+- Add webpack dev server.
+- Split webpack config into separate files for each env.
+- Add webpack bundle analyzer/visualizer.
```

Comment:
Can you add a note in the readme about how/when this would be used?

Reply:
Assuming you meant the webpack bundle visualizations, done.